doc/source/how-to/dns-configuration.rst

```diff
@@ -0,0 +1,93 @@
+DNS configuration
+=================
+
+As explained in :ref:`Documenting`, PyAnsys projects publish their documentation
+online under the following canonical name (CNAME) convention:
+
+``https://<product>.docs.pyansys.com``
+
+In order to request a CNAME for the ``pyansys.com`` domain, please contact
```

Comment:
```suggestion
To request a CNAME for the ``pyansys.com`` domain, please contact
```

Reply:
I tend to use "in order to..." too. However, as per @PipKat reviews, I know most of the times its usage can be avoided as it does not introduce any information.